app/src/project/project.reducers.js

```diff
@@ -3,7 +3,7 @@ import * as giveTimeActions from './components/giveTime/giveTime.actionTypes'
 import * as projectRowActions from './components/projectRow/projectRow.actionTypes'
 import * as addProjectActions from './components/addProject/addProject.actionTypes'
 
-export default function (state = { projects: [] }, action) {
+export default function (state = { projects: [], snackbar: { open: false, message: '' } }, action) {
```

Comment:
Why does the snackbar state leaks here ? 
It's inside a project specific reducer, whe should only care about projects